modules/wallpapers/Angelus/ReadMe.txt

```diff
@@ -0,0 +1,55 @@
+Angelus' Wallpapers
+===================
+Forum User "Angelus" (jedi.knight@wanadoo.es)
+Under Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License.
+See forum thread: https://reactos.org/forum/viewtopic.php?f=25&t=6153
+
+Bliss Series: 1; 2; 3 (Cristal ROS); 4 (DreamBliss)
+-------------
+Photos of tree, cloud, sky and grass taken separately in a park near where the author lives.
+
+BlueHorizon
+-----------
+Photo (blue filtered) taken from Santiago de Compostela looking west.
+
+DeepSea
+-------
+An underwater ambiance.
+
+(Lonely) Fisherman
+------------------
+Photography taken at Corrubedo, a beautiful place in the northwest of Spain.
+
+Flower (WaterLily)
+------------------
+Photo of water lily using the HDR technique with 3 shots of -1/0/+1 EV.
+
+Rain in the Screen
+------------------
+Photo was taken from the author's room window in a raining afternoon.
+
+ReactOS
+-------
+Inspired from the new OS from Redmond. There are four versions: darker, lighter, blueish and greenish.
+
+ReactOS - A New Hope
+--------------------
+
+ReactOS Sunset
+--------------
+The photo was taken in a colorful sunset where the author lives. We can see the new Earth's satellite ;-)
+
+Sea Series
+----------
+1 (Golden Sunset): Beautiful sunset in the Atlantic ocean.
+2: Photo taken at Praia do Vilar (Vilar's Beach) in the Barbanza region.
+3 (Red Sunset): Photo taken from Carril, a beautiful village at the port town of Vilagarc�a de Arousa.
```

Comment:
Mojibake.

Reply:
Make the file UTF-8? [Vilagarcía de Arousa](https://en.wikipedia.org/wiki/Vilagarc%C3%ADa_de_Arousa)

Reply:
```suggestion
3 (Red Sunset): Photo taken from Carril, a beautiful village at the port town of Vilagarcía de Arousa.
```